apps/web/components/WishlistPageContent/WishlistPageContent.vue

```diff
@@ -71,6 +77,8 @@ import { productGetters, productImageGetters } from '@plentymarkets/shop-api';
 import { SfLoaderCircular } from '@storefront-ui/vue';
 import type { WishlistPageContentProps } from '~/components/WishlistPageContent/types';
 
+const runtimeConfig = useRuntimeConfig();
+const showNetPrices = runtimeConfig.public.showNetPrices;
```

Comment:
im not sure if this is a temporarily solution but in general the showNetPrice should be dynamic

Reply:
This should be temporary only, I created another PBI to properly fix this:

https://dev.azure.com/plentymarkets/plentymarkets/_workitems/edit/132247